src/index.ts

```diff
@@ -0,0 +1,2 @@
+const Kinvey = require('kinvey-phonegap-sdk');
```

Comment:
Why require instead of import? Is it some sort of typings issue?

Reply:
The `kinvey-phonegap-sdk` uses `module.exports` to export the SDK. If I use `import` then I get undefined. I must use `require`.